docker/packager/packager

```diff
@@ -253,8 +253,8 @@ def parse_env_variables(
 
     if sanitizer:
         result.append(f"SANITIZER={sanitizer}")
-    if build_type:
-        result.append(f"BUILD_TYPE={build_type.capitalize()}")
+    if debug_build:
+        result.append("BUILD_TYPE=DEBUG")
```

Comment:
put my comment in commit by accident
This breaks tests because in `clickhouse-test` we check for `Debug` when processing tags
Can we make this `Debug`?

Reply:
Sure, it must be `Debug`

```
In [1]: "debug".capitalize()
Out[1]: 'Debug'
```

I misinterpreted `capitalize` and `upper`